aten/src/ATen/native/quantized/cpu/kernels/QuantizedOpKernels.cpp

```diff
@@ -2089,6 +2090,33 @@ void fake_quantize_tensor_cachemask_kernel(
       }
     });
   });
+  }
+
+void fake_quantize_tensor_cachemask_kernel(
+    Tensor& output,
+    Tensor& mask,
+    const Tensor& input,
+    float sc,
+    int64_t z_point,
+    int64_t quant_min,
+    int64_t quant_max) {
+  _fake_quantize_tensor_helper(output, mask, input, sc, z_point, quant_min, quant_max);
+
+}
+
+void fake_quantize_tensor_cachemask_tensor_qparams_kernel(
+    Tensor& output,
+    Tensor& mask,
+    const Tensor& input,
+    const Tensor& sc,
+    const Tensor& z_point,
+    int64_t quant_min,
+    int64_t quant_max) {
+
+  float* scale_ptr = sc.data_ptr<float>();
+  int32_t* zp_ptr = z_point.data_ptr<int32_t>();
+
+  _fake_quantize_tensor_helper(output, mask, input, *scale_ptr, *zp_ptr, quant_min, quant_max);
```

Comment:
is this trying to get the per tensor scale parameter? I think we can use sc.item() since it's a scalar